pages/index.js

```diff
@@ -2,7 +2,7 @@ import { useState } from 'react';
 import Filter from '../components/Filter/index.js';
 import Header from "../components/Header/index.js";
 import Question from "../components/Question/index.js";
-import Button from "../components/Button"
+import Button from "../components/Button/index.js";
```

Comment:
Heads up just in case it was forgotten, (thought it doesn't look like it)
When you name the file index.js, you don't have to specify the file in the path.